api/v1/tests.py

```diff
@@ -1,3 +1,68 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse_lazy
+
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from fixtureless.factory import create
+
+from recipes.models import Recipe
+
+
+class TestRecipeApi(APITestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create(username='test_user')
+        self.recipe = create(Recipe, {'title': 'testing'})
+
+    def test_recipe_list(self):
+        """
+        Verify GET returns Recipe list
+        """
+        response = self.client.get(reverse_lazy('api:v1:recipe-list'))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
getting the status code is great, but if we're actually testing what this method indicates, that the view returns a recipe list, you should check the contents of the response body too and make sure the json list is there that you expect

Reply:
checking the status code should be part of every test that calls an endpoint or even a view, but it's only the lowest hanging fruit, a default minimum that the test should meet. Then you need to test the actual functionality you expect to exist